src/ViewHelpers/StagesPageViewHelper.php

```diff
@@ -95,7 +95,7 @@ public static function displayStages(array $stages): string
                 $result .= '<td class="col-xs-2 text-center"><a class="toggleEditForm">Edit</a></td>';
                 if (empty($stage->getOptions()) && $stage->getHasAssignees() === 0) {
                     $result .= '<td class="col-xs-2 text-center"><a data-hasOptions="0" data-id="'
-                        . $stage->getStageId() . '" class="text-danger delete disabled">Delete</a></td>';
+                        . $stage->getStageId() . '" class="text-danger delete">Delete</a></td>';
```

Comment:
This code is all fine BUT your pull request is doing two completely different things here: the documentation of some routes and also, unrelatedly, fixing the issue with the delete button on the application stages page.

Ideally, these would be two separate pull requests, but I'm happy to merge them together. Just be aware not to clump together unrelated things in the future!